### PR TITLE
[graph] JSON format reference

### PR DESCRIPTION
**Pull request by franramirez688**
Docs for https://github.com/conan-io/conan/pull/13934 https://github.com/conan-io/conan/pull/13964 https://github.com/conan-io/conan/pull/13967

### CODE REVIEW

File: reference/commands/graph/info.rst [hunk 1]
```diff
@@ -356,9 +350,340 @@ For example, to get the options of zlib, the following command could be run:
...
+                },
+                "generators": [],
+                "system_requires": {},
+                "recipe_folder": "/Users/franchuti/.conan2/p/binut53bd9b3ee9490/e",
+                "source_folder": null,
+                "build_folder": null,
+                "generators_folder": null,
+                "package_folder": null,
+                "cpp_info": {
+                    "root": {...}
```
Comment (franramirez688) [inline reference/commands/graph/info.rst:579]: I've collapsed these ones

File: reference/commands/graph/info.rst [hunk 2]
```diff
@@ -356,9 +350,340 @@ For example, to get the options of zlib, the following command could be run:
...
+                "context": "host",
+                "test": false
+            }
+        },
+        "root": {
+            "0": "None"
+        },
+        "overrides": {},
+        "resolved_ranges": {}
+    }
```
Comment (franramirez688) [inline reference/commands/graph/info.rst:680]: Full output or should I collapse more dicts?

Comment (czoido) [reply reference/commands/graph/info.rst:680]: As the format is the same for several commands, like create, install, graph... I think we should make a separate section to document the format and link it in each of the commands?

Comment (memsharded) [reply reference/commands/graph/info.rst:732]: I agree, there should be a section with this json file format, that is linked by all the different commands.

**Review by memsharded — APPROVED** [review]
Looks good!

File: reference/commands/create.rst [hunk 3]
```diff
@@ -140,6 +133,10 @@ the package has been created correctly. Check :ref:`testing Conan packages
 
         $ conan create . --test-folder=
 
+
+.. include:: ./graph/graph_info_format_json.inc
```
Comment (memsharded) [inline reference/commands/create.rst:137]: Not an inclusion, but a cross-reference

File: reference/commands/export-pkg.rst [hunk 4]
```diff
@@ -77,11 +77,8 @@ The ``conan export-pkg`` command creates a package binary directly from pre-comp
 
 In general, it is expected that when ``conan export-pkg`` executes, the possible Conan dependencies that were necessary to build this package had already been installed via ``conan install``, so it is not necessary to download dependencies at ``export-pkg`` time. But if for some reason this is not the case, the command defines ``--remote`` and ``--no-remote`` arguments, similar to other commands, as well as the ``--skip-binaries`` optimization that could save some time installing dependencies binaries if they are not strictly necessary for the current ``export-pkg``. But this is the responsibility of the user, as it is possible that such binaries are actually necessary, for example, if a ``tool_requires = "cmake/x.y"`` is used and the ``package()`` method implements a ``cmake.install()`` call, this will definitely need the binaries for the dependencies installed in the current machine to execute.
 
-.. warning::
-
-  The json output of the ``conan export-pkg --format=json`` is **experimental** and subject to
-  change.
+.. include:: ./graph/graph_info_format_json.inc
```
Comment (memsharded) [inline reference/commands/export-pkg.rst:80]: cross-ref is better

**Review by memsharded — CHANGES_REQUESTED** [review]
Sorry, my bad. Cross-referencing will be better than extra duplication.

**Review by RubenRBS — APPROVED** [review]

File: reference/commands/graph/info_format_json.rst [hunk 5]
```diff
@@ -0,0 +1,229 @@
+.. _reference_commands_graph_info_json_format:
```
Comment (memsharded) [inline reference/commands/graph/info_format_json.rst:1]: Problem is that files need to be referenced in the TOC, not only cross-referenced

Alternatives are:
Reference->Commands->New "output" section
Reference->Extensions->API->New "output" section

Where "output" can be "Formatters output", our "Output files format", or something like that, that explains the graph.html, the graph.json, the ``pkg-list.json`` once it is released, etc.

**Review by memsharded — APPROVED** [review]

**Comment by franramirez688** [comment]
Superseded by https://github.com/conan-io/docs/pull/3236 (bad base branch)

**Pull request closed without merging**